core/src/main/scala/org/apache/spark/storage/MemoryStore.scala

```diff
@@ -333,15 +322,6 @@ private[spark] class MemoryStore(blockManager: BlockManager, memoryManager: Memo
     blockId.asRDDId.map(_.rddId)
   }
 
-  private def tryToPut(
```

Comment:
I dropped this overload so that it's easier to grep around the code in IntelliJ.